tests/test_feature_engineering.py

```diff
@@ -0,0 +1,32 @@
+import operator
+
+import ibis
+import pytest
+
+import ibisml as ml
+
+
+@pytest.fixture()
+def train_table():
+    N = 100
+    return ibis.memtable({"x": list(range(N)), "y": [10] * N, "z": ["s"] * N})
+
+
+def test_PolynomialFeatures(train_table):
+    step = ml.PolynomialFeatures(ml.numeric(), degree=2)
+    step.fit_table(train_table, ml.core.Metadata())
+    result_table = step.transform_table(train_table)
+    sol = train_table.mutate(
+        **{
+            "poly_x^2": operator.pow(train_table.x, 2),
+            "poly_x_y": operator.mul(train_table.x, train_table.y),
+            "poly_y^2": operator.pow(train_table.y, 2),
```

Comment:
```suggestion
            "poly_x^2": train_table.x ** 2,
            "poly_x_y": train_table.x * train_table.y,
            "poly_y^2": train_table.y ** 2,
```
Much easier to read. :)